media-libs/rtaudio/rtaudio-5.2.0.recipe 

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A set of C++ classes that provide a common API for realtime audio input/output"
+DESCRIPTION="This distribution of RtAudio contains the following:
+
+doc: RtAudio documentation (see doc/html/index.html)\
+tests: example RtAudio programs\
+include: header and source files necessary for ASIO, DS & OSS compilation\
+tests/Windows: Visual C++ .net test program workspace and projects"
+HOMEPAGE="http://www.music.mcgill.ca/~gary/rtaudio/"
+COPYRIGHT="Gary P. Scavone, 2001-2022 (and many other developers!)"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/thestk/rtaudio/archive/refs/tags/5.2.0.tar.gz"
+CHECKSUM_SHA256="a8d9c738addffd485c3f0bab14cbba72600267e3113f274398c67829bbb49332"
+SOURCE_DIR="rtaudio-$portVersion"
+PATCHES="rtaudio-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	librtaudio$secondaryArchSuffix
+	lib:librtaudio$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	"
+
+PROVIDES_devel="
+	devel:librtaudio$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	rm -r build
+	mkdir -p build && cd build
+	cmake .. $cmakeDirArgs -DBUILD_SHARED_LIBS=ON  -DCMAKE_BUILD_TYPE=Release
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make -C build install
+	#mv $portDir/build/data/cmake/rtaudio $dataDir/cmake/Modules/rtaudio
```

Comment:
these commented out lines should be removed if the problem was fixed elsewhere (I think it is fixed in the patch?)